puppet/modules/chassis/manifests/openssl.pp

```diff
@@ -0,0 +1,28 @@
+define chassis::openssl (
+) {
+	file { '/vagrant/certs':
+		ensure => 'directory',
+	}
+
+	# include ::openssl
+
+	# class { '::openssl':
+	  # package_ensure         => latest,
+	  # ca_certificates_ensure => latest,
+	# }
+	
+	$commonname = "*.${fqdn}"
+
+	openssl::certificate::x509 { $fqdn:
+	  country      => 'HM',
```

Comment:
I'd rather we use a [private code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#User-assigned_code_elements) here; `AA` or `ZZ` sound preferable.